subsys/bluetooth/controller/Kconfig

```diff
@@ -107,8 +108,8 @@ config SYSTEM_WORKQUEUE_STACK_SIZE
 
 config BT_CTLR_SDC_LLPM
 	bool "Enable Low Latency Packet Mode support"
-	depends on SOC_SERIES_NRF52X
 	select BT_CONN_PARAM_ANY if !BT_HCI_RAW
+	depends on (SOC_SERIES_NRF52X || SOC_SERIES_NRF54LX || SOC_SERIES_NRF54HX)
```

Comment:
I wonder if it would be more future-proof to have `depends on (!SOC_NRF5340_CPUNET)` or similar